CommentingPractice/comment-practice.js

```diff
@@ -5,12 +5,16 @@
 import Template from './model/template.js';
 import * as amonte from './services/capitalizer.js';
 
+import * as luis from './services/capitalizer.js';
```

Comment:
Here you're importing from the wrong file. You want to import from './services/backwardsName.js'